alibi_detect/od/pytorch/mahalanobis.py

```diff
@@ -0,0 +1,113 @@
+from typing import Optional, Union
+
+import torch
+
+from alibi_detect.od.pytorch.base import TorchOutlierDetector
+
+
+class MahalanobisTorch(TorchOutlierDetector):
+    ensemble = None
+
+    def __init__(
+            self,
+            min_eigenvalue: float = 1e-6,
+            device: Optional[Union[str, torch.device]] = None
+            ):
+        """PyTorch backend for Mahalanobis detector.
+
+        Parameters
+        ----------
+        min_eigenvalue
+            Eigenvectors with eigenvalues below this value will be discarded.
+        device
+            Device type used. The default None tries to use the GPU and falls back on CPU if needed.
+            Can be specified by passing either ``'cuda'``, ``'gpu'`` or ``'cpu'``.
+        """
+        TorchOutlierDetector.__init__(self, device=device)
```

Comment:
Is there a reason we've not used `super()` here like everywhere else in library?